.github/workflows/main.yaml

```diff
@@ -45,28 +45,40 @@ jobs:
             artifact_name: 'wasmer-linux-amd64'
             cross_compilation_artifact_name: 'cross_compiled_from_linux'
             run_integration_tests: true
+            use_sccache: true
           - build: macos-x64
             os: macos-latest
             llvm_url: 'https://github.com/wasmerio/llvm-custom-builds/releases/download/11.x/darwin-amd64.tar.gz'
             artifact_name: 'wasmer-darwin-amd64'
             cross_compilation_artifact_name: 'cross_compiled_from_mac'
             run_integration_tests: true
+            use_sccache: false
           - build: macos-arm64
             os: macos-11.0
             target: aarch64-apple-darwin
             artifact_name: 'wasmer-darwin-arm64'
+            use_sccache: false
           - build: windows-x64
             os: windows-latest
             # llvm_url: 'https://github.com/wasmerio/llvm-custom-builds/releases/download/11.x/windows-amd64.tar.gz'
             artifact_name: 'wasmer-windows-amd64'
             cross_compilation_artifact_name: 'cross_compiled_from_win'
             run_integration_tests: true
+            use_sccache: false
```

Comment:
We probably want this to be true!

Reply:
I suspect that sccache isn't doing anything on most of our platforms, just testing it out now

Reply:
Windows is the only one that I'm more unsure. On the rest I think it should shave significant time.

Reply:
Alright, you're right! Sccache definitely makes them all faster. Glad I checked though!